src/bp/bpfs.ts

```diff
@@ -113,7 +114,7 @@ class BPFS {
       if (useForce) {
         console.info(chalk.blue(`Force pushing local changes to ${this.serverUrl}...`))
 
-        await axiosClient.post('update', archive)
+        await axiosClient.post('update', archive, { timeout: ms('20m') })
```

Comment:
Is there any reason why the timeout is of 20 minutes instead of let's say 10 or 15 minutes?

Reply:
It is 20 minutes in the frontend API call, so I preferred to use the same

![image](https://user-images.githubusercontent.com/13484138/116469380-780a8880-a848-11eb-8f05-0d37d2cfecf6.png)